fluid-python/output.py

```diff
@@ -0,0 +1,77 @@
+import os
+
+
+
+def writeOutputToVTK(time, name, dx,  data, datanames):
+
+    if type(data) is not list:
+        data = list(data)
+    if type(datanames) is not list:
+        datanames = list(datanames)
+
+    n_datasets = data.__len__()
+    assert n_datasets == datanames.__len__()
+
+    outpath = os.path.join(os.getcwd(), 'Postproc')
```

Comment:
I would prefer to name all directories starting with a lowercase, actually (for consistency with the rest, and for easy guessing/tab-completion):
```suggestion
    outpath = os.path.join(os.getcwd(), 'postproc')
```
(similarly, update the path and any other scripts)